inst/schema/PopulationResponseData.schema.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "maxItems": 0
```

Comment:
Is this the most appropriate way to assert we're not returning any data? We could use `null` here but there is some problem with the conversion from an R `NULL` into json `null`.

It can be done by passing `null = "null"` to `jsonlite::toJSON`. Which is fine but it means our `errors` section on a success would become null whereas at the moment we're relying on the fact that `jsonlite::toJSON` converts that to an empty object `{}`.

So the options I can see are
* Have errors and population data as an empty object
* Update them both to null in the schema and pass `null = "null"` to `jsonlite::toJSON`
* Pass `null = "null"` to `jsonlite::toJSON` but manually set errors to a named empty list for a success `response` (meaning we don't have to change Response schema - it looks like a bit of a faff to create a named empty list though)
* Pass `null = "null"` to `jsonlite::toJSON` but manually set errors to an empty list and update its type in schema to `array` for a `success` response (this makes it consistent with `failure` response which is already an array)

or something else I haven't thought of.